### ISSUE TITLE
Criteria of TSC Represented Projects?

### ISSUE THREAD

**Issue by AkihiroSuda** [moby/tsc#1]
https://github.com/moby/tsc/tree/4e9cd33666f71c55283a00f59ec78b76d525938a#tsc-represented-projects

It might be good to state clearly:
- Criteria of TSC Represented Projects
- How TSC represented / non-represented project can be added / deleted
- How addition / deletion of TSC represented project affects the number of the existing committee seats

cc @crosbymichael @thaJeztah

**Comment by crosbymichael** [comment]
A lot of these details will be figured out after the TSC has been elected.  We just need to bootstrap the process right now then this group can take it from there.

**Comment by AkihiroSuda** [comment]
Yes, however, I think some basic explanation about the criteria before bootstrapping would be appreciated for ensuring legitimacy of the bootstrap committee.

For example, why runc is not TSC represented while containerd is TSC represented?

**Comment by tianon** [comment]
I imagine that's probably because runc is already governed entirely by the OCI now, and as such is not going to be governed as part of the "Moby" umbrella (even if it ends up being considered a Moby project -- the OCI is responsible for determining the governance of runc).

**Comment by AkihiroSuda** [comment]
How does it differ from CNCF containerd?

**Comment by thaJeztah** [comment]
Containerd still used the BDSL model; https://github.com/containerd/containerd/blob/3596ec53ab24046aaa67da2d392857fc888a6078/MAINTAINERS#L119-L148

**Comment by cpuguy83** [comment]
CNCF does not enforce a governance model on its projects.

**Comment by AkihiroSuda** [comment]
Any progress on this?
